templates/patterns/featured-item/featured_item.ui_patterns.yml

```diff
@@ -0,0 +1,50 @@
+featured_item:
+  label: "Featured item"
+  description: "The featured item component uses a different visualisation to attract user attention. It's always the first item on the \"Latest section\"."
+  variants:
+    '':
```

Comment:
Is this the correct way, to leave empty for default? I'm asking because I don't know.

Reply:
I don't see other more correct solution. Yml is still valid. I did this strictly by documentation.
But probably `~` should work as well and actually have more nicer view.
Next reviewer I hope will help with this dilemma.

Reply:
I also had a similar issue on my own PR and I don't have a clear answer either. I would try with ~ just because it seems cleaner but I'll leave it to @sergepavle